packages/gatsby-theme-docs/src/layouts/internals/layout-release-note.js

```diff
@@ -59,7 +66,9 @@ ReleaseNote.propTypes = {
   description: PropTypes.string.isRequired,
   type: PropTypes.oneOf(['feature', 'enhancement', 'fix']).isRequired,
   topics: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
-  body: PropTypes.string.isRequired,
+  body: PropTypes.string,
```

Comment:
Since you're no longer querying for `body` from `src/templates/release-notes-list.js`, this prop is no longer necessary.

Reply:
The component is used on the details page, too so it needs to be able to accept either a full body or just an excerpt. 

But I could refactor it in a way that the excerpt is passed as the body and in addition we pass just the boolean that tells which renderer to use.

Reply:
> But I could refactor it in a way that the excerpt is passed as the body and in addition we pass just the boolean that tells which renderer to use.

Yes that would be preferrable. This component should not concern itself where it's rendered, the parent knows the context.

Reply:
@davifantasia  let's talk about this one in our half hour today - I'm confused about my own implementation now, maybe it has to do with work you did on the details pages. 

There isn't any second use of this component any more in the codebase.

Reply:
See https://github.com/commercetools/commercetools-docs-kit/pull/375#discussion_r411267318